src/Templates/parts/nav.twig

```diff
@@ -10,6 +10,7 @@
       </button>
       {% endif %}
       <a class="navbar-brand" href="{{baseurl}}">MyRadio</a>
+      <a class="navbar-brand" href="{{homeurl}}">ury.org.uk</a>
```

Comment:
Not really much point in making it configurable if you just hardcode the text anyway?